### ISSUE TITLE
readthedocs

### ISSUE THREAD

**Issue by lemmatum** [PlasmaPy/SpectroscoPy#3]
Setup readthedocs site.

**Comment by StanczakDominik** [comment]
Okay, I'm not sure why this isn't working. Some issue with versioning, as seen [in the logs](https://readthedocs.org/projects/spectroscopy/builds/6239101/).

```
Running Sphinx v1.6.5
making output directory...

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/spectroscopy/conda/latest/lib/python3.6/site-packages/sphinx/cmdline.py", line 305, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/spectroscopy/conda/latest/lib/python3.6/site-packages/sphinx/application.py", line 168, in __init__
    confoverrides or {}, self.tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/spectroscopy/conda/latest/lib/python3.6/site-packages/sphinx/config.py", line 150, in __init__
    execfile_(filename, config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/spectroscopy/conda/latest/lib/python3.6/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 91, in <module>
    version = package.__version__.split('-', 1)[0]
AttributeError: module 'SpectroscoPy' has no attribute '__version__'

Exception occurred:
  File "conf.py", line 91, in <module>
    version = package.__version__.split('-', 1)[0]
AttributeError: module 'SpectroscoPy' has no attribute '__version__'
The full traceback has been saved in /tmp/sphinx-err-__ewy91w.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

I'm not quite certain why it isn't pulling the version variable from `setup.cfg`, or whether it should do that, even.

**Comment by lemmatum** [comment]
Looks like the [conf.py](https://github.com/PlasmaPy/PlasmaPy/blob/master/docs/conf.py) that PlasmaPy is using is way different from the default from astropy template. The version number is manually defined in that one. The astropy template [documentation](http://docs.astropy.org/en/latest/development/affiliated-packages.html) says to change the version number in setup.py. I'll try that out.

**Comment by lemmatum** [comment]
No luck 😞

**Comment by lemmatum** [comment]
So I ran through these scripts by hand... it looks like `setup.py` pulls in the version from `setup.cfg` just fine. It is also able to write the version to `SpectroscoPy/version.py`. Yet, when I import SpectroscoPy and call `__version__` it is indeed missing! So there is either something about `setup.py` or the project structure, maybe `__init__.py` files, which results in the package missing the version attribute. `conf.py` then fails for obvious reasons.

**Comment by lemmatum** [comment]
And we're in [business](http://spectroscopy.readthedocs.io/en/latest/)!
Solved by #14
